package.json

```diff
@@ -46,8 +47,10 @@
     "node-uuid": "^1.4.7",
     "numeral": "^1.5.3",
     "q": "~1.4.1",
+    "session-file-store": "0.2.0",
```

Comment:
I think this was accidentally added back in.  Could you remove session file store?